feat/feature_test.go

```diff
@@ -269,16 +289,12 @@ func (s *S) TestBaseOrientationOf(c *check.C) {
 
 func (s *S) TestOrientationWithin(c *check.C) {
 	for _, t := range orientationTests {
-		c.Check(feat.OrientationWithin(t.f, t.ref), check.Equals, t.ori)
-	}
-
-	// Check we can find things that are all along the feature chain.
-	for ref := feat.Feature(pribA); ref != nil; ref = ref.Location() {
-		c.Check(feat.OrientationWithin(pribA, ref), check.Equals, feat.Forward)
+		c.Check(feat.OrientationWithin(t.f, t.ref), check.Equals, t.oriWithin)
 	}
 
 	// Check that unorientable features return different reference features.
 	c.Check(feat.OrientationWithin(pribA, nil), check.Equals, feat.NotOriented)
+	c.Check(feat.OrientationWithin(orfB, chrom1), check.Equals, feat.NotOriented)
```

Comment:
I think this is wrong. The intuition here is that a chromosome is intrinsically not orientable since it has no location, however regions on that chromosome *are* orientable with respect to that chromosome. It is true that orfB is not universally orientable, but that is not what this is asking.